web_session_allow_public/README.rst

```diff
@@ -0,0 +1,56 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
```

Comment:
Good call, I need to start using Bob templates or something so I stop getting these copy paste errors. Will run a find/replace